astroplan/core.py

```diff
@@ -2,6 +2,21 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
+from astropy.coordinates import (EarthLocation, Latitude, Longitude, SkyCoord,
+                                 AltAz)
+import astropy.units as u
+from astropy.units.Quantity import Quantity
```

Comment:
ImportError, should be:
```python
form astropy.units import Quantity
```
You can always check locally whether most things are OK by running
```
python2 setup.py test
python3 setup.py test
```